### ISSUE TITLE
isInDST() incorrect in some situations

### ISSUE THREAD

**Issue by hawkett** [moment/luxon#908]
**Describe the bug**
`isInDST()` can return an incorrect value when daylight time starts in January.

For example `DateTime.fromISO('1974-01-11T05:00', { zone: 'America/Los_Angeles' })` has the correct `offsetNameLong` (Pacific Daylight Time), and correct `offset` (-420), but `isInDst()` incorrectly returns `false`.

It looks like the Luxon code reasonably assumes that a change like this would not happen in January. In 1974 it occurs on Jan 6, so the offset is the same in both January and May.

```
get isInDST() {
  if (this.isOffsetFixed) {
    return false;
  } else {
    return (
      this.offset > this.set({ month: 1 }).offset || this.offset > this.set({ month: 5 }).offset
    );
  }
}
```

**To Reproduce**
```
const dt = DateTime.fromISO('1974-01-11T05:00', { zone: 'America/Los_Angeles' });

console.log(dt.offsetNameLong); // 'Pacific Daylight Time' (correct)
console.log(dt.offset);         // -420 (correct)
console.log(dt.isInDST);        // false (incorrect)
```

**Comment by icambron** [comment]
Yeah, would accept a fix for this

**Comment by socraticDevBlog** [comment]
I've submited a fix: https://github.com/moment/luxon/pull/1199

**Comment by icambron** [comment]
Fixed in #1199
